src/App.tsx

```diff
@@ -15,71 +17,88 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export class App extends React.Component {
+  state = {
+    selectedGood: 'Jam',
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  handleAddButton = (condition: boolean, newGood: string) => {
```

Comment:
As for me, duplicate:
```
const isSelected = selectedGood === good;
```
will be more preferable instead pass `condition` argument